bpf/flows.c

```diff
@@ -42,6 +41,17 @@
 #define INGRESS 0
 #define EGRESS 1
 
+#define FIN_FLAG 0x1
+#define SYN_FLAG 0x2
+#define RST_FLAG 0x3
+#define URG_FLAG 0x4
+#define PSH_FLAG 0x5
+#define ECE_FLAG 0x6
+#define CWR_FLAG 0x7
+#define SYN_ACK_FLAG 0x8
+#define FIN_ACK_FLAG 0x9
+#define RST_ACK_FLAG 0xa
```

Comment:
Flags need to be powers of 2, so each one set their own exclusive bit
```suggestion
#define FIN_FLAG 1
#define SYN_FLAG 2
#define RST_FLAG 4
#define URG_FLAG 8
#define PSH_FLAG 16
#define ECE_FLAG 32
#define CWR_FLAG 64
#define SYN_ACK_FLAG 128
#define FIN_ACK_FLAG 256
#define RST_ACK_FLAG 512
```